splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -508,6 +507,44 @@ def generate(
         logger.info(
             f"Installed add-on requirements into {ucc_lib_target} from {source}"
         )
+        if global_config.has_custom_search_commands():
+            for command in global_config.custom_search_commands:
+                file_path = os.path.join(source, "bin", command["fileName"])
+                if not os.path.isfile(file_path):
+                    logger.error(
+                        f"{command['fileName']} is not present in `{os.path.join(source, 'bin')}` directory. "
+                        "Please ensure the file exists."
+                    )
+                    sys.exit(1)
+
+                if (command["requireSeachAssistant"] is False) and (
+                    command.get("description")
+                    or command.get("usage")
+                    or command.get("syntax")
+                ):
+                    logger.warning(
+                        "requireSeachAssistant is set to false "
+                        "but atrributes required for 'searchbnf.conf' is defined which is not required."
+                    )
+                if (command["requireSeachAssistant"] is True) and not (
+                    command.get("description")
+                    and command.get("usage")
+                    and command.get("syntax")
+                ):
+                    logger.error(
+                        "One of the attributes among `description`, `usage`, `syntax` "
+                        " is not been defined in globalConfig. Defined them as requireSeachAssistant is set to True. "
+                    )
+                    sys.exit(1)
+                if command["version"] == 1:
```

Comment:
Should we not support version 1? We mention in this PR that it is deprecated.

Reply:
Though version 1 is deprecated, we should support it as add-ons like ServiceNow and BMC Remedy uses version 1 for their custom search commands, hence I kept it.

Reply:
Let's remove it from UCC, I'll create a Jira for the team to move to the version 2.

Reply:
Btw, I checked both those add-ons and I can't find `version` in `commands.conf`. Is there another way to figure out which version do they use?

Reply:
The determination of version is can be seen based on [this description](https://conf.splunk.com/files/2017/slides/extending-spl-with-custom-search-commands-and-the-splunk-sdk-for-python.pdf#page=20).

Okay, I'll remove the support of version 1 from this PR.

Reply:
Removed the support of version 1.